com.unity.render-pipelines.high-definition/Documentation~/Reflection-Probe-Usage.md

```diff
@@ -0,0 +1,44 @@
+# Reflection probe usage
+
+## What is a probe
+
+Reflection probes and planar reflection probes provide indirect specular lighting.
+
+A probe capture its surroundings and will influence specular lighting in a specific volume.
+A reflection probe will capture all of its surroundings while a planar reflection probe will capture only one direction.
+
+## Controlling the influence of a probe
+
+The influence of a probe determines which pixels will be affected and by how much.
+
+There are 3 tools to control the influence on a pixel:
+* Influence volume: any pixel inside this volume will be affected.
+* Blend distance: pixel near the border of the volume will be less influenced.
+* Blend normal distance: pixels near the border with an invalid normal won't be affected.
```

Comment:
* [Blend normal distance](#blend-normal-influence): The probe does not affect pixels near the border with an invalid normal.